app/ui/node-settings/browser.js

```diff
@@ -0,0 +1,310 @@
+(function () {
+  'use strict'
+  const handlebars = require('handlebars')
+  const uuid4      = require('uuid4')
+  const i18n       = require('../../lib/i18n.js')
+
+  const logger       = require('../../lib/logger.js');
+  const clientConfig = require('../../lib/config.js');
+  const syncFactory  = require('@gyselroth/balloon-node-sync');
+
+  const electron = require('electron');
+  const ipcRenderer = electron.ipcRenderer;
+
+  handlebars.registerHelper('i18n', (key) => {
+    var translation = i18n.__(key);
+
+    return new handlebars.SafeString(translation);
+  })
+
+  var sync,
+      promise = new Promise((resolve) => {
+        if (!clientConfig.get('authMethod')) {
+          logger.info('AUTH: no authentication method set yet');
+          return resolve();
+        }
+
+        clientConfig.retrieveSecret(clientConfig.getSecretType()).then((secret) => {
+          clientConfig.setSecret(secret);
+          resolve();
+        }).catch((error) => {
+          logger.error('AUTH: failed retrieve secret from keystore', {error});
+          resolve();
+        });
+      });
+
+  promise.then(() => {
+    initNodeSettings();
+  });
+
+  function initNodeSettings() {
+    sync = syncFactory(clientConfig.getAll(true), logger);
+
+    $(document).ready(() => {
+      var localNode = sync.lstatSync(clientConfig.get('nodePath'))
+      sync.find({ino: localNode.ino}, (err, syncedNode) => {
+        if (!syncedNode) {
+          return;
+        }
+
+        if (syncedNode.length) {
+          sync.blnApi.getAttributes({id: syncedNode[0].remoteId, useId: true}, ['id', 'path', 'meta.tags', 'changed'], (err, data) => {
+            logger.info('attributes: ' + JSON.stringify(data))
```

Comment:
* Log a complete phrase.
* Add data to context object (And do not stringify context data)
* Add a category

See example: https://github.com/gyselroth/balloon-client-desktop/blob/master/app/lib/auth/controller.js#L33

Reply:
And review all other log requests in this pr